ApolloCodegen/Package.swift

```diff
@@ -13,7 +13,7 @@ let package = Package(
         .package(name: "Apollo",
                  url: "https://github.com/apollographql/apollo-ios.git",
                  /// Make sure this version matches the version in your iOS project!
-                 from: "1.0.0-alpha-1"),
+                 from: "1.0.0-alpha.1"),
```

Comment:
```suggestion
                 from: "1.0.0-alpha.2"),
```

Good catch and thanks for the contribution! We can bump this to `.2` since that's the latest alpha release and it added some important functionality re. operation variables and field arguments.

_Keep an eye out for alpha release 3 coming out soon!_